scriptoocloud/src/main/java/edu/asu/diging/scriptoocloud/web/forms/DatasetEditForm.java

```diff
@@ -0,0 +1,35 @@
+package edu.asu.diging.scriptoocloud.web.forms;
+
+import javax.validation.constraints.NotEmpty;
+
+public class DatasetEditForm {
+
+    private Long id;
+    private int index;
```

Comment:
what is this for?

Reply:
A Page of Datasets are displayed in the view.  The index variable allows the error to be displayed adjacent to the correct Dataset in the list using Thymeleaf's IterStat.